Feature/dynamic attributes accordion menu (#315)
CoherentLabs/GameUIComponents: components/accordion-menu/README.md

|---|---|---|---|
|disabled  | Boolean   |false   | If the panel is disabled. You can't expand or shrink disabled panels  |
|expanded   | Boolean   |false   | If the panel is expanded on load    |

### v3.1.0

You can change the attributes mentioned above runtime via `setAttribute`, `removeAttribute` or with JavaScript.

For example we can change the `multiple` of accordion menu like that
```javascript
const accordionMenu = document.querySelector('gameface-accordion-menu');
// If we want to enable multiple of the accordion menu
accordionMenu.setAttribute('multiple', ''); 
// or
accordionMenu.multiple = true;
// And if we want to disable the multiple of the accordion menu
accordionMenu.removeAttribute('multiple');
// or
accordionMenu.multiple = false;
```

> **Not that when disabling the `multiple` functionality of the accordion-menu and there are more than one accordion-panels expanded then it will close all of them if are not disabled and leave just the first one expanded.**

Reviewer: ```suggestion
**Note that when disabling the `multiple` functionality of the accordion-menu and there are more than one accordion-panels expanded then it will close all of them if are not disabled and leave just the first one expanded.**
```

Author: Fixed

> You can change the `disabled` and `expanded` of the accordion-panel in the simmilar way as above.

Reviewer: ```suggestion
You can change the `disabled` and `expanded` attributes of the accordion-panel in the similar way as above.
```

Author: fixed

```javascript
const accordionPanel = document.querySelector('gameface-accordion-panel');
// If we want to disable the accordion panel
accordionPanel.setAttribute('disabled', ''); 
// or
accordionPanel.disabled = true;
// And if we want to enable the accordion panel
accordionPanel.removeAttribute('disabled');
// or
accordionPanel.disabled = false;
// If we want to expand the accordion panel
accordionPanel.setAttribute('expand', ''); 
// or

> accordionPanel.expand = true;

Reviewer: ```suggestion
accordionPanel.expanded = true;
```

Author: Fixed

// If we want to collapse the accordion panel

> accordionPanel.removeAttribute('expand'); 

Reviewer: ```suggestion
accordionPanel.removeAttribute('expanded'); 
```

// or

> accordionPanel.expand = false;

Reviewer: ```suggestion
accordionPanel.expanded = false;
```

Author: Fixed
